src/System.Diagnostics.StackTrace/src/System.Diagnostics.StackTrace.csproj

```diff
@@ -27,6 +32,18 @@
   <ItemGroup>
     <Compile Include="System\Diagnostics\StackFrameExtensions.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == ''">  
+    <Compile Include="System\Diagnostics\StackTraceSymbols.CoreCLR.cs" />  
```

Comment:
I guess I only waiting for feedback from Wes on this issue. I’ve addressed got an ok from the rest of the reviewers.

From: Matt Ellis [mailto:notifications@github.com]
Sent: Wednesday, May 18, 2016 6:26 PM
To: dotnet/corefx <corefx@noreply.github.com>
Cc: Mike McLaughlin <mikem@microsoft.com>; Assign <assign@noreply.github.com>
Subject: Re: [dotnet/corefx] Add symbol support helper class for mscorlib's StackTrace. (#8670)


In src/System.Diagnostics.StackTrace/src/System.Diagnostics.StackTrace.csproj<https://github.com/dotnet/corefx/pull/8670#discussion_r63810461>:

> @@ -27,6 +32,18 @@

>    <ItemGroup>

>      <Compile Include="System\Diagnostics\StackFrameExtensions.cs" />

>    </ItemGroup>

> +  <ItemGroup Condition="'$(TargetGroup)' == ''">

> +    <Compile Include="System\Diagnostics\StackTraceSymbols.CoreCLR.cs" />

@weshaggard<https://github.com/weshaggard> This project compiles for three target groups. I think we only need this for .NET Core (since CoreCLR will use it) but can you confirm this is the right thing to do?

—
You are receiving this because you were assigned.
Reply to this email directly or view it on GitHub<https://github.com/dotnet/corefx/pull/8670/files/46ebb77cdef1f2c23a599e5f0dbd2e0cb7fa9d59#r63810461>